tests/karate/src/test/java/be/utils/JsonConverter.java

```diff
@@ -0,0 +1,92 @@
+package be.utils;
+
+
+import com.intuit.karate.Json;
+
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Base64;
+
+public class JsonConverter {
+
+  public Json fromMapToJson(Map<String,String> map){
+    return Json.of(map);
+  }
+
+  private String senderPk = "J9fBzJV70Jk5c-i3277Uq4CmeL4t53WDfUghaK0HpeM=";
+  private String signature = "ONylxgHA9cbsB_lwdfbn3iyzRd4aTpJhBMnvEKhmJF_niE_pUHdmjxDXjEwFyvo5WiH1NZXWyXG27SYEpkasCA==";
+  private String messageIdForced = "";
+
+  /*
+    Produces the base64 variant of the json file passed as argument
+   */
+  public String convertJson(Json json){
+    String stringJson = json.toString();
+    byte[] jsonBytes = stringJson.getBytes(StandardCharsets.UTF_8);
+    Base64.Encoder encoder = Base64.getEncoder();
+    String base64 = encoder.encodeToString(jsonBytes);
+    return base64;
+  }
+
+  /*
+    Produces a valid Json representation of a message given the message data, the id of the message
+    and the channel where the message is supposed to be sent
+   */
+  public Json messageFromData(String stringData, String method, int id, String channel){
```

Comment:
I am not very convinced by this function name.

From the implementation, you can only create publish messages and broadcast messages.

And in the current implementation, you only produces publish messages.

Why not droping the method argument and going with `createPublish`

The call would look like
```
jsonConverter.createPubish(data, id, channel)
```
instead of
```
jsonConverter.messageFromData(data, "publish", id, channel)
```

I personally find it easier to understand. What do you think ?